tests/phpunit/tests/rest-api/rest-settings-controller.php

```diff
@@ -83,13 +83,60 @@ public function test_context_param() {
 	public function test_get_item_is_not_public_not_authenticated() {
 		$request  = new WP_REST_Request( 'GET', '/wp/v2/settings' );
 		$response = rest_get_server()->dispatch( $request );
-		$this->assertSame( 401, $response->get_status() );
+		$data     = $response->get_data();
+		$actual   = array_keys( $data );
+
+		$expected = array(
+			'description',
+			'language',
+			'page_for_posts',
+			'page_on_front',
+			'show_on_front',
+			'title',
+		);
+
+		sort( $expected );
```

Comment:
```suggestion
		// Array should be in alphabetical order.
		$expected = array(
			'description',
			'language',
			'page_for_posts',
			'page_on_front',
			'show_on_front',
			'title',
		);

```

Nitpick: The sort isn't necessary. I think a comment suffices.